### ISSUE TITLE
online use of condor

### ISSUE THREAD

**Issue by JackYu1234-dot** [FXIhub/condor#40]
I am a graduate student from China. I think condor is a very excellent scientific research tool. However, the online web version of condor is very unstable. It often cannot be opened or errors occur. I don't know if it's a problem with my network or a problem with the condor server.

**Comment by FilipeMaia** [comment]
Yeah, this is often a problem. We'll now start to have automated monitoring of the website.
If you do get errors feel free to post an issue.
